### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by LeningWang** [Doubiiu/DynamiCrafter#128]
How does the loss function behave, and after how many steps train can we generate an image instead of noise? Thanks!

**Comment by Doubiiu** [comment]
Hi, since we build our model based on pre-trained t2v model VideoCrafter. it can already generate videos at the beginning.

**Comment by LeningWang** [comment]
Thank you! If I use the base architecture of DynamiCrafter to train on our own dataset without using the existing pre-trained parameters, is that feasible? Approximately how many steps would the training take? Thanks again!

**Comment by Doubiiu** [comment]
Emmm I am not sure. Usually, we will train models on pre-trained weights to make our life easier.
